src/handle_connect.c

```diff
@@ -13,6 +13,7 @@ and the Eclipse Distribution License is available at
 Contributors:
    Roger Light - initial implementation and documentation.
 */
+/* Portions copyright © 2018 TIBCO Software Inc. */
```

Comment:
you added three lines of code? really?

Reply:
gotta love big company legal processes ...

I'm hoping they will get better about open source project support soon, but at the moment I just happy to be allowed to submit :)